docs/Using-the-dataset-validator.md

```diff
@@ -421,25 +428,24 @@ Validation of study succeeded with warnings.
 ```
 
 ## Validation of non-human data ##
-When importing a study, the validator assumes by default that the following parameters from `portal.properties` are set to:
+When importing a study, the validator assumes by default that the following three parameters 
+`--species` , `--reference_genome` , `--genome_build` are set to the following:
 ```
-species=human
-ncbi.build=37
-ucsc.build=hg19
+--species human
```

Comment:
I don't think it hurts to have `=` signs - the arg parser should interpret the arguments the same way with or without `=` signs

Reply:
(this is also not necessary)

Reply:
changed.